container/kubernetes/onboarding/README.md

```diff
@@ -88,16 +88,23 @@ CreateDeployments:
        	GitTag: <MACHINE_CONFIG_TAG_OR_COMMITID_OR_BRANCH>
 ```
 6. Initiate the Deployment process based on three actions; create, update or delete.
+   AND gatewaydeployment yaml file path.
```

Comment:
I don't think this is needed, the commands clearly show that the yaml file is required.

Reply:
will remove it.